blocks/links-bar-block/package.json

```diff
@@ -30,7 +31,8 @@
     "@arc-fusion/prop-types": "^0.1.5",
     "@wpmedia/engine-theme-sdk": "*",
     "@wpmedia/news-theme-css": "*",
```

Comment:
we want to remove the packages like news-theme-css that are not being used per the checklist. Shared styles should also be removed
```suggestion
```